config/locales/en.yml

```diff
@@ -1524,30 +1530,4 @@ en:
         no_hours_listed: Sorry, no hours listed.
         no_phone_number_listed: Sorry, no phone number listed.
         phone_number_title: Call %{provider_name}
-        search_radius: Within %{distance} miles of %{zip} (%{zip_name})
-    zendesk:
-      extracts:
-        filename: Filename
-        header: Anonymized Intake CSV Extracts
-        record_count: Record Count
-        run_at: Run At
-      intakes:
-        banking_info:
-          account_number: Account number
-          account_type: Account type
-          bank_name: Bank name
-          routing_number: Routing number
-      sign_in:
-        cta: Sign in with Zendesk
-        title: First, we need you to verify with Zendesk!
-      tickets:
-        show:
-          chart:
-            bank_info: Bank Info
-            consent: Consent
-            doc_type: Doc Type
-            drop_off_document_bundle: Document Bundle
-            header: The following chart does not include documents uploaded by VITA volunteers or client support
-            link: Link
-            notes: Notes
-            upload_date: Upload Date
+        search_radius: Within %{distance} miles of %{zip} (%{zip_name})
```

Comment:
Very minor, non-blocking, FYI: If the PR retained the end-of-file newline, then this line would have not been shown in the diff. I configure RubyMine to add end-of-file newlines to avoid diff noise like this.

Reply:
Sorry about that. I'm happy to add that same configuration. Could you show me sometime?